ledger_device_sdk/src/ui/gadgets.rs

```diff
@@ -361,15 +365,16 @@ impl<'a> Menu<'a> {
     }
 }
 
-#[derive(Copy, Clone, PartialEq)]
+#[derive(Copy, Clone, PartialEq, Default)]
 pub enum PageStyle {
+    #[default]
     PictureNormal, // Picture (should be 16x16) with two lines of text (page layout depends on device).
-    PictureBold,   // Icon on top with one line of text on the bottom.
-    BoldNormal,    // One line of bold text and one line of normal text.
-    Normal,        // 2 lines of centered text.
+    PictureBold, // Icon on top with one line of text on the bottom.
+    BoldNormal,  // One line of bold text and one line of normal text.
+    Normal,      // 2 lines of centered text.
 }
 
-#[derive(Copy, Clone)]
+#[derive(Copy, Clone, Default)]
 pub struct Page<'a> {
```

Comment:
As I understand `chunk_count` and `chunk_idx` are now useless fields in the `Page` struct as the `Field` manages the display of this information by itself. 

The logic in the `Page` `place` function that manages the display of chunk and chunk count for `PageStyle::BoldNormal` should be removed.

Reply:
Done